src/core/Akka.Streams/Dsl/Sink.cs

```diff
@@ -270,8 +270,7 @@ public static class Sink
         /// </summary>
         /// <typeparam name="TIn">TBD</typeparam>
         /// <returns>TBD</returns>
-        public static Sink<TIn, Task> Ignore<TIn>()
-            => new Sink<TIn, Task>(new SinkholeSink<TIn>(Shape<TIn>("BlackholeSink"), DefaultAttributes.IgnoreSink));
+        public static Sink<TIn, Task> Ignore<TIn>() => FromGraph(new IgnoreSink<TIn>());
```

Comment:
This method should return Sink<TIn, Task<Done>>
See https://github.com/akka/akka/blob/master/akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala#L197